File: FE/lib/models/post.dart
class Post {
  final String profileImageUrl;
  final String username;
  final String time;
  final String content;
  final String likes;
  final String comments;
  final String shares;

  Post(
      {required this.profileImageUrl,
      required this.username,
      required this.time,
      required this.content,
      required this.likes,
      required this.comments,
      required this.shares});
}

File: BE/app/models/post.py
class Post: 
    def __init__(self, content : str, user_id : str, files = [], comments = [], likes = []):
        self.content = content
        self.files = files
        self.comments = comments
        self.likes = likes
        self.user_id = user_id

    @staticmethod
    def to_dict(self):
        return vars(self)
    def from_dict(source):
        pass
    def __repr__(self) -> str:
        return(
            f'Account(\
                content={self.content}, \
                _id={self.user_id}, \
                files={self.files}, \
            )'
        )